src/community/jdbc-metrics/pom.xml

```diff
@@ -1,67 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unrelated to this specific file, I've found that jdbc-metrics is still included in the docker instructions:

```
> git grep "jdbc-metrics"
doc/en/user/source/installation/docker.rst:       cog-http                      jdbc-metrics                    sec-keycloak
```

Reply:
There is now a separate commit for changes to common documentation files.